members/admin/activityparticipant_admin.py

```diff
@@ -227,10 +226,13 @@ def get_queryset(self, request):
         return qs.filter(activity__department__adminuserinformation__user=request.user)
 
     def activity_person_gender(self, item):
-        if item.person.gender == "MA":
-            return "Dreng"
-        elif item.person.gender == "FM":
-            return "Pige"
+        if item.person.gender is not None:
+            if item.person.gender == "MA":
+                return "Dreng"
+            elif item.person.gender == "FM":
+                return "Pige"
+            else:
+                return item.person.gender
         else:
             return "Andet"
```

Comment:
I stedet for alt det her logik, hvis vi skal have Andet, kan vi så ikke defineret andet som den enum det er?

Altså definere det som OT eller noget i den stil